tasks/main.yml

```diff
@@ -37,20 +38,22 @@
 
 - name: Set NTP server addresses
   purefa_ntp:
-    state: "{{ purefa_ntp_state | default('present')}}"
+    state: "{{ 'present' if purefa_ntp_servers else 'absent' }}"
```

Comment:
Is the if/else needed? if `purefa_ntp_servers ` isn't defined, this task won't run anyway, due to the `when` on line 45

Reply:
It is needed because the user might want to define the variable `purefa_ntp_servers` as empty and therefore clear out the current values.

Reply:
Ah I see. Makes sense.